### PR TITLE
Bedgraph bedclip bedgraphtobigwig

### LINKED ISSUE #2194: new subworkflow bedgraph_to_bigwig

### PR DESCRIPTION
**Pull request by KamilMaliszArdigen**
This is migrated rnaseq subworkflow bedgraph_to_bigwig

## PR checklist

Closes #2194  

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Review by matthdsm — APPROVED** [review]
LGTM!

File: subworkflows/nf-core/bedgraph_bedclip_bedgraphtobigwig/meta.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  - bedgraph:
+      type: file
+      description: bedGraph file after clipping
+      pattern: "*.bedGraph"
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+authors:
+  - "@KamilMaliszArdigen"
```
Comment (Emiller88) [inline subworkflows/nf-core/bedgraph_bedclip_bedgraphtobigwig/meta.yml:32]:
```suggestion
  - "@drpatelh"
  - "@KamilMaliszArdigen"
``` 

I believe @drpatelh wrote it in rnaseq. I think authors is who's responsible for maintaining it.

**Review by Emiller88 — CHANGES_REQUESTED** [review]
Just updating the authors. Otherwise LGTM!

File: subworkflows/nf-core/bedgraph_bedclip_bedgraphtobigwig/meta.yml (hunk 1 above)
Comment (KamilMaliszArdigen) [reply subworkflows/nf-core/bedgraph_bedclip_bedgraphtobigwig/meta.yml:32]: True missed that.

**Review by Emiller88 — COMMENTED** [review]
It looks good, but the tests are still failing :disappointed: Do you know why?

**Comment by KamilMaliszArdigen** [comment]
Tests for different subworkflow are failing unfortunately :( 
'tests/subworkflows/nf-core/fastq_align_dna/test.yml'

**Review by Emiller88 — APPROVED** [review]
Whoo!

**Pull request merged**